### PR TITLE
Hello Stimulus :) 

### PR DESCRIPTION
**Pull request by ytbryan**
Will you want this?

Added a modified hello stimulus example from the `stimulusjs/stimulus` repo. 

Note that it dynamically add the html code to the dom to simplify the usage for hello stimulus. You can see it at `hello_stimulus.js`. 

cc @javan

### CODE REVIEW

File: lib/install/examples/stimulus/hello_stimulus.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import { Application } from "stimulus"
+import { autoload } from "stimulus/webpack-helpers"
+
+const application = Application.start()
+const context = require.context("./controllers", true, /\.js$/)
+autoload(context, application)
+
+//You may replace the code below with its equivalent html code 
+document.addEventListener('DOMContentLoaded', () => {
```
Comment (ytbryan) [inline lib/install/examples/stimulus/hello_stimulus.js]: kinda ugly but I think it simplifies the usage?

**Comment by javan** [comment]
Thanks! But let's hold off until Stimulus hits 1.0. It's only been public for two days and the documentation is incomplete.

**Comment by xdmx** [comment]
Shall this be reopened/merged now that stimulus 1.0 has been released? 🙄

**Comment by ytbryan** [comment]
Will be happy to reopen and modify the PR according to 1.0 changes

**Comment by Abumuhd** [comment]
Ok..doned 😎

**Pull request closed without merging**